src/scss/_main.scss

```diff
@@ -1,4 +1,4 @@
 body {
-  width: 100%;
```

Comment:
Dlaczego taka zmiana? ;)
Dasz radę wytłumaczyć?

Reply:
W sumie pomyślałem, że to jest zbędne i powinno zostać dodane przy okazji robienia tła.